app/assets/stylesheets/admin_rtl.scss

```diff
@@ -0,0 +1 @@
+@import "admin";
```

Comment:
Sorry if I'm missing something but isn't this file just similar to `admin.scss`?

Reply:
Yes. `admin_rtl.scss` imports `admin.scss` which imports `common/admin/admin_base.scss` which in turn has a ton of imports and rules that are specific to admin pages.

Our stylesheet compiler expects an identically-named file under `app/assets/stylesheets` for each core bundle, see:

https://github.com/discourse/discourse/blob/d10a050da22d9fc315b8b7ea5e572e1262649c8e/lib/stylesheet/compiler.rb#L25-L39

If we wanted, I think we could special-case bundles that end with `_rtl`, but I think keeping things clear/explicit is a good things plus this allows us to add RTL-specific CSS rules if we need to do that in the future.

Reply:
How does adding a `_rtl` postfix at the end adds the RTL-specific CSS rules? :thinking: Do we have a check somewhere which looks for the postfix?

Reply:
If we add a rule or an import inside that `admin_rtl.scss` file, then that rule or import will be present in the RTL bundle and not the LTR bundle.

Reply:
But there is nothing in the `admin_rtl.scss` file except for `@import "admin"` which is the same as `admin.scss`. Is this PR to prepare for changes that we are going to introduce in the future? I'm also curious how `.pull-left{float:right}` is injected into the RTL scss files.

Reply:
Yes, there's no RTL-specific stuff in `admin_rtl.scss` at the moment, but keeping this entry point file for the RTL bundle allows us to do that if we need to. For example, most of the rules in this file:

https://github.com/discourse/discourse/blob/d10a050da22d9fc315b8b7ea5e572e1262649c8e/app/assets/stylesheets/common/base/rtl.scss#L1-L83

are rules that only need to exist in the RTL bundle and not the LTR bundle, but they currently exist in both. With some refactoring, I can fix that so those rules only exist in the RTL bundle, but we need to have a separate file for the RTL bundle that's not the same as the LTR bundle's entry file. We do the same for the mobile and desktop bundle, i.e. we have `mobile_rtl.scss` and `desktop_rtl.scss`.

Reply:
Got it :+1: So this PR will make it easier for us to move RTL into the RTL specific files in the future.

Reply:
Yes, this PR creates RTL versions for the admin bundle (and plugins), and also allows us to have RTL-specific rules in the RTL version without having them in the LTR versions as well.